src/test/integration/suite/test-utils.ts

```diff
@@ -0,0 +1,38 @@
+import retry from "async-retry";
+import { EnvironmentModel } from "../../../get-environments";
+import * as vscode from "vscode";
+import sinon from "sinon";
+import { Credentials } from "../../../types";
+
+export const waitFor = <T>(callback: (...args: any[]) => T): Promise<T> => {
+  return retry(callback, { retries: 5, minTimeout: 300, maxTimeout: 1000 });
+};
+
+export const login = async (auth: Credentials) => {
+  const inputStub = sinon.stub(vscode.window, "showInputBox");
+  inputStub.onFirstCall().resolves(auth.keyId);
+  inputStub.onSecondCall().resolves(auth.secret);
+  await vscode.commands.executeCommand("env0.login");
```

Comment:
Same as clicking on login button in the welcome view, since the login button in welcome view is just executing this command